modules/generators/content/password_file/templates/password_file.md.erb

```diff
@@ -1,14 +1,13 @@
 # Password list
 
 <%
-    website_count = 1
-    website_sample.each { |wsite|
+    website_count = 0
+    pass_list.each { |pass|
```

Comment:
You can use each_with_index to improve your code:
https://apidock.com/ruby/Enumerator/each_with_index